components/CollaboratorsAmount/SubmitForm.tsx

```diff
@@ -7,23 +8,37 @@ const SubmitForm = () => {
   const { setTab, collaboratorsAmount, setCollaboratorsAmount } =
     useContractBuilderProvider()
 
+  const handleCollaboratorsChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value
+
+    if (value === '') {
+      setCollaboratorsAmount(0)
+
+      return
+    }
+
+    if (isNaN(parseInt(e.target.value, 10))) return
```

Comment:
what would happen if this was `simplified`???
```suggestion
    if (isNaN(parseInt(e.target.value, 10))) {
      setCollaboratorsAmount(0)
      return
    }
```